src/test/processmessage_tests.cpp

```diff
@@ -82,7 +82,7 @@ struct DummyXThinProcessor : public XThinBlockProcessor {
         BlockHeaderProcessor& h) : XThinBlockProcessor(f, w, h), misbehaved(0)
     { }
 
-    virtual void misbehave(int howmuch) {
```

Comment:
Ah so THAT's what the override kwd is good for!

Reply:
`override` has saved me from making errors like this so many times... but no this time :cry: